### PR TITLE
link to Geo Engine in VAT help

### PR DESCRIPTION
**Pull request by michaelmattig**

### CODE REVIEW

**Comment by michaelmattig** [comment]
[Geo Engine im VAT verlinken](https://trello.com/c/xbFGwidi/2059-geo-engine-im-vat-verlinken)

File: projects/dashboards/gfbio/src/app/help/help.component.html [hunk 1]
```diff
@@ -1,11 +1,18 @@
...
-    cooperation with the Senckenberg Biodiversity and Climate Research Centre (BiK-F) (head: Prof. Thomas Hickler). You can contact us at
-    <a href="mailto:vat@mathematik.uni-marburg.de" target="_blank">vat@mathematik.uni-marburg.de</a> or visit our
-    <a href="https://github.com/umr-dbs/VAT " target="_blank">GitHub</a> for further information.
-</p>
+<div class="attribution">
+    <p>
+        <span class="vat">Vat</span> is developed by the Database Research Group of the University of Marburg (head: Prof. Bernhard Seeger)
+        in cooperation with the Senckenberg Biodiversity and Climate Research Centre (BiK-F) (head: Prof. Thomas Hickler). You can contact
+        us at <a href="mailto:vat@mathematik.uni-marburg.de" target="_blank">vat@mathematik.uni-marburg.de</a> or visit our
+        <a href="https://github.com/umr-dbs/VAT " target="_blank">GitHub</a> for further information.
```
Comment (ChristianBeilschmidt) [inline projects/dashboards/gfbio/src/app/help/help.component.html:8]: This info and repo are outdated.

File: projects/dashboards/gfbio/src/app/help/help.component.html [hunk 2]
```diff
@@ -1,11 +1,18 @@
...
+    </p>
+    <p>
+        <span class="vat">Vat</span> is built upon the Geo Engine, a cloud-ready geo-spatial data processing platform. Learn more about Geo
+        Engine on <a href="https://github.com/geo-engine" target="_blank">GitHub</a> or visit the Geo Engine
+        <a href="https://www.geo-engine.de" target="_blank">Website</a>.
```
Comment (ChristianBeilschmidt) [inline projects/dashboards/gfbio/src/app/help/help.component.html:13]:
```suggestion
        <a href="https://www.geo-engine.de" target="_blank">website</a>.
```

**Review by ChristianBeilschmidt — CHANGES_REQUESTED** [review]

File: projects/dashboards/gfbio/src/app/help/help.component.html (hunk 1 above)
Comment (michaelmattig) [reply projects/dashboards/gfbio/src/app/help/help.component.html:8]: the repo is outdated, true. But what about the rest? It's still devleoped/maintained by agdbs right? Do you want me to remove Senckenberg as well?

Comment (ChristianBeilschmidt) [reply projects/dashboards/gfbio/src/app/help/help.component.html:8]: The instance is maintained by AGDBS, right.

But I would link to the new repo and I'm not sure that this e-mail works.

I'm also not sure if BiK-F needs still to be there, maybe there could be some kudos that they helped in the design.

Comment (michaelmattig) [reply projects/dashboards/gfbio/src/app/help/help.component.html:8]: The E-Mail works. It somehow lands in my Uni mailbox. But maybe we should link to the gfbio.org impressum instead?

Comment (ChristianBeilschmidt) [reply projects/dashboards/gfbio/src/app/help/help.component.html:8]: Ok

Comment (michaelmattig) [reply projects/dashboards/gfbio/src/app/help/help.component.html]: 4e57f06720143f30f3f3caaeaddc3c630d8f8d35

**Review by ChristianBeilschmidt — APPROVED** [review]

**Pull request merged**